src/main/scala/cognite/spark/v1/ThreeDModelsRelation.scala

```diff
@@ -11,7 +11,7 @@ import org.apache.spark.sql.{Row, SQLContext}
 final case class ModelItem(id: Long, name: String, createdTime: Long)
 
 class ThreeDModelsRelation(config: RelationConfig)(val sqlContext: SQLContext)
-    extends SdkV1Relation[ThreeDModel, Long](config, "threeDModels.read") {
```

Comment:
this shortName was different for once. I think this is only used for metric so the change should have no impact.